scripts/zones/VeLugannon_Palace/mobs/Detector.lua

```diff
@@ -77,18 +77,7 @@ end;
 -----------------------------------
 
 function onMobDeath(mob, player, isKiller)
-
     checkGoVregime(player,mob,743,1);
-
-    local Detector = mob:getID();
-    local Caretaker = mob:getID() + 1;
-
-    GetMobByID(Detector):resetLocalVars();
-
-    if (GetMobAction(Caretaker) ~= 0) then
-        DespawnMob(Caretaker);
-    end
-
```

Comment:
the same code was in there twice, once under onMobDeath and once under onMobDespawn